### PR TITLE
Added Travis CI

### PR DESCRIPTION
**Pull request by MiguhRuiz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
+script: npm run postinstall
+script: node index.js
```
Comment (jenkoian) [inline .travis.yml]: How does this work in travis environment? As it just loads the server and waits for input, I'm not sure how travis knows when it's complete?

**Comment by jenkoian** [comment]
Thanks for this @MiguhRuiz I think we should add some basic tasks like linting (using es lint or something). Also thinking it may be worth building against a few versions of node, perhaps current LTS (4.6 I think) and latest stable (6.7 I think).

File: .travis.yml (hunk 1 above)
Comment (MiguhRuiz) [inline .travis.yml]: Per default it runs `npm install` and then `npm test`but if you dont have any test you might configure something like I did.

**Comment by jenkoian** [comment]
Cheers for this, have updated the app for 2017 so closing this, feel free to re-submit! 😃

**Pull request closed without merging**